ocrd_anybaseocr/cli/ocrd_anybaseocr_layout_analysis.py

```
@@ -165,6 +215,7 @@ def process(self):
                 """ % model_path)
             sys.exit(1)
```

This should be `LOG.raise()` instead of `LOG.error()`+`sys.exit()`.

Besides, it is unacceptable to do:
```python
if not tf.test.is_gpu_available():
    LOG.error("Your system has no CUDA installed. No GPU detected.")
    sys.exit(1)
```

TF is perfectly capable of running on CPU, and GPU should never be a hard requirement for OCR-D.

---

so far we were not aware that GPU should not be a hard requirement. in previous meetings we specifically asked which hardware we can assume to be available

---

ok, maybe I got this wrong. Better ask officially. But from a user perspective, I think you should always allow running on CPU only as well (even if this will take 10 times as long), especially if it's possible by default on your framework.